$NetBSD$

Add note that GUI test programs are not executed automatically.

--- test/Makefile.orig	2020-07-16 10:25:51.000000000 +0000
+++ test/Makefile
@@ -195,6 +195,10 @@ GLALL = \
 	shape$(EXEEXT)
 
 all:	$(ALL) $(GLDEMOS)
+	echo "-------------------------------------------"
+	echo 'Test programs were built in $${WRKSRC}/test.'
+	echo "You have to start them manually."
+	echo "-------------------------------------------"
 
 gldemos:	$(GLALL)
 
